README.md

```diff
@@ -87,3 +87,21 @@ python scripts/calc_gt_coco.py
 ```
 
 Set the dataset and split parameters in the top section of the script.
+
+## Manual annotation tool
+
+To annotate a new dataset in BOP format use [this tool](./scripts/annotation_tool.py).
```

Comment:
Please write down the python command with required arguments instead

Reply:
done